src/PQDashboard/Views/Main/Home.cshtml

```diff
@@ -536,9 +536,11 @@
         return false;
     }
 
-    function openFaultDetailsByDateXDAPage(){
-        if(contextfromdate == contexttodate){
-            window.open(xdaInstance + "/Workbench/FaultsDetailsByDate.cshtml?MeterIds=" + meterList.selectedIdsString() + "&Date=" + contexttodate + "&context=" + globalContext)
+    function openFaultDetailsSEBrowser(){
```

Comment:
Can you double check the timewindow for this? I am assuming this should be a single day? you may need to skip the `time` piece and hardcode it to noon since 2pm +/- 12 hours won't give you a day (it would be 2am to 2am instead).

Reply:
Noted, I'll go ahead and switch it around